src/Components/footer/Footer.js

```diff
@@ -1,9 +1,19 @@
 import React, { Component } from "react";
+import './Footer.scss';
+
+const footerList = ['FALCON 9', 'FALCON HEAVY', 'DRAGON', 'STARSHIP', 'SHOP', 'ACCOUNT'];
 
 class Footer extends Component {
   render() {
-    return <div></div>;
+    return (
+      <div className='footer'>
+        <span>SPACEX © 2020</span>
+        <ul className='footerUl'>
+          {footerList.map((el, index) => <li key={index}><a href='#'>{el}</a></li>)}
```

Comment:
```suggestion
          {footerList.map((el, index) => <li key={index}>
            <a href='#'>{el}</a>
          </li>)}
```
부모 자식 관계가 잘 드러날 수 있도록 펼쳐서 작성해주세요

Reply:
수정하겠습니다